week11-functional-programming-redux/redux-qa.md

```diff
@@ -0,0 +1,40 @@
+## Redux Q&A##
+
+Q: Does anyone use Redux without React out there? Examples of such projects?
+- it’s not common, but you can do it
+- both are great for dealing with states
+- Redux sort of evolved alongside React, but its inspiration came from Elm
+- Redux is completely pure, there are no dependencies (unlike React)
+
+Q: When would you not use React?
```

Comment:
Do you mean, 'when would you not use Redux?' or was it really React that was meant here?